createContacts/function.json

```diff
@@ -0,0 +1,26 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": ["get", "post"]
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    },
+    {
+      "type": "cosmosDB",
+      "direction": "out",
+      "name": "outputDocument",
+      "databaseName": "my-database",
+      "collectionName": "my-container1",
+      "createIfNotExists": false,
```

Comment:
what is the purpose of this property?

Reply:
when a change is imposed on a specific container name and the container doesn’t exist a new container will be created if this value is set to true